oaknational/Oak-Web-Application
# chore: update mv for downloads and overview - LESQ-674

## Description

Music year: 1967

- Connects the MV's that use 
```
  {
            "copyright_info": "Attributions in slides where needed."
  }
```
For both legacy and new content to determine if a lesson has downloads
More detail here - https://www.notion.so/oaknationalacademy/Spring-Sprint-1-fe3f5ac1bb0147668eb40dc32334d684?p=aa1063242dc847a2b0cb5eb96788b104&pm=s

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2293--oak-web-application.netlify.thenational.academy
teachers/programmes/science-primary-ks2/units/simple-forces-including-magnets/lessons/different-magnets-and-their-parts
 You should see downloads are available 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
